content/10.diagnostics.md

```diff
@@ -28,6 +30,13 @@ In this case the test was designed to detect sarbecoviruses, of which SARS-CoV-2
 Considering that no other sarbecoviruses are currently known to infect humans, a positive test indicates that the patient is infected with SARS-CoV-2.
 However, this test is not able to discriminate the genetics of viruses within the sarbecovirus clade.
 
+#### Pooled PCR Testing Approaches
+
+Due to limited supplies and the need for more tests, a number of labs have found ways to enable pooled or similar tests.
```

Comment:
```suggestion
Due to limited supplies and the need for more tests, a number of labs have found ways to enable pooled or otherwise parallelized tests.
```
I am also not sure of exactly what the right word would be! Does "parallelized" do it? I was also thinking "multiplexed"?

Reply:
Multiplex PCR is something different, isn't it? How about "group testing"? What would be examples of these other pool-like tests?